src/TH1Plotter.cc

```diff
@@ -861,16 +861,22 @@ namespace plotIt {
     for (size_t i = 0; i < first_bin; i++) {
       underflow += h->GetBinContent(i);
       underflow_sumw2 += (h->GetBinError(i) * h->GetBinError(i));
-      h->SetBinContent(i, 0); // Clear bin content so that Integral() still returns the right value
     }
-
     float overflow = 0;
     float overflow_sumw2 = 0;
     for (size_t i = last_bin + 1; i <= (size_t) h->GetNbinsX() + 1; i++) {
       overflow += h->GetBinContent(i);
       overflow_sumw2 += (h->GetBinError(i) * h->GetBinError(i));
-      h->SetBinContent(i, 0); // Clear bin content so that Integral() still returns the right value
     }
+    // Clear out-of-range bin content so that Integral() still returns the right value
+    for (size_t i = 0; i < first_bin-1; i++) {
```

Comment:
`i = 0` is still the underflow bin, maybe start at 1. And why `first_bin - 1` instead of `first_bin`? From a quick look, it seems the bin `first_bin - 1` is not cleared, but it was previously.